src/ZipFile.jl

```diff
@@ -487,6 +500,9 @@ end
 
 # Write nb elements located at p into f.
 function write(f::WritableFile, p::Ptr, nb::Integer)
+    if nb == 0
+        return 0
+    end
```

Comment:
It seems like a zlib or Zlib.jl bug. Maybe switching to Libz.jl will fix this (see #22).

For now, you can send a separate PR with a comment saying this is a workaround zlib/Zlib.jl bug.